### PR TITLE
Expand Notch page and re-org to make clearer

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-imu-notch-filtering.rst [hunk 1]
```diff
@@ -6,37 +6,48 @@
...
 
 With the introduction of dynamic notch filtering, the need for static notch filtering is reduced, although it can be useful in targeting particular resonant frequencies (e.g. of the frame). We will therefore look at dynamic filtering first.
 
+Key to the dynamic notch filter operation is control of its center frequency. There are five methods that can be used for doing this:
+
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 0. Dynamic notch disabled. The center frequency is fixed and is static. Often used in Traditional Helicopters with external governors for rotor speed, either incorporated in the ESC or separate for ICE motors.
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1. (Default) Throttle position based, where the frequency at mid-throttle is determined by analysis of logs, and then variation of throttle position above this is used to track the increase in noise frequency.
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:22]: Actually disabled is the default

File: common/source/docs/common-imu-notch-filtering.rst [hunk 2]
```diff
@@ -6,37 +6,48 @@
...
 
+--------------------
 Throttle-based Setup
-====================
-
-If you do not have ESC telemetry then throttle-based setup is generally the way to go.
+--------------------
 
-- Set :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1 to use a throttle-based estimation to set the harmonic notch frequency.
+If you do not have ESC telemetry or an RPM sensor, then throttle-based (:ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1 ) center frequency control is best.
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:48]: Not sure about best, FFT is generally better and easier to set up if you have the CPU cycles

**Review by andyp1per — APPROVED** [review]

File: common/source/docs/common-imu-notch-filtering.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-imu-notch-filtering.rst:48]: will change appropriately

File: common/source/docs/common-imu-notch-filtering.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-imu-notch-filtering.rst:22]: yeah, we need to add the INS_HNTCH_ENABLE param at the top...will do

**Pull request merged**